labonneboite/scripts/create_index.py

```diff
@@ -414,7 +415,15 @@ def create_offices_for_departement(departement):
 
     logger.info("STARTED indexing offices for departement=%s ...", departement)
 
-    for _, office in enumerate(db_session.query(Office).filter_by(departement=departement).all()):
+    for _, office in enumerate(db_session.query(Office).filter(
```

Comment:
Je sais que cette ligne ne fait pas partie de la PR, mais à quoi sert le `enumerate` ?

Reply:
bonne question: `TypeError: 'Office' object is not iterable`

il est bien nécéssaire

Reply:
Euh tu es sûr ? J'arrive à écrire `for office in db_session.query(Office).all(): print(office)`

Reply:
ah oui au temps pour moi j'avais oublié de dégager le `_` dans mon test

ça a l'air good, je prends